charter.md

```diff
@@ -1,134 +1,159 @@
+ - IAPS/INAF: Institute for Space Astrophysics and Planetology 
+ - BOM: Bureau of Meteorology
+ - PRIC: Polar Research Institute of China
+ - ISEE: Institute for Space-Earth Environmental Research
+ - IRAP: Research Institute in Astrophysics and Planetology 
+ - NICT: National Institute of Information and Communications Technology
+ - DICE/UEC: University of Electro-Communications
+ - SDU: Shandong University
+ - UNIS: The University Centre in Svalbard
+ - UKZN: University of KwaZulu-Natal
+ 
 ## Purpose
 
-This working group (WG) is charged with maintaining the protocols for transferring RAWACF and DAT 
-files from the PI institutions, first to a holding institution (Virginia Tech) and ultimately to
-a distribution institution (University of Saskatchewan) where they are merged and copied onto hard 
-drives for distribution. The cadence for distribution of the drives is every four months 
-(Jan-Apr, May-August, September-December). This working group was created after the SuperDARN 
-workshop in Moose Jaw, Saskatchewan in 2013. The group is also charged with maintaining an 
-augmentation to the hard drive distribution, any networked SuperDARN Data Mirrors.
+This WG is charged with maintaining the protocols for transferring Level 1 data 
+files from PI institutions, first to holding institutions and 
+ultimately to distribution institutions where they are merged, checked, catalogued and copied to 
+servers for community access. 
 
-## Holding Institution
+## Holding Institution(s)
 
 The holding institution prior to October 1<sup>st</sup>, 2013 was JHU/APL. See Appendix A 
 for a description of the system that was in place at JHU/APL.
 
 The holding institution beginning October 1<sup>st</sup>, 2013 is Virginia Tech. 
 See Appendix B for a description of the system in place at Virginia Tech.
 
+An additional holding institution was put in place in 2016 at BAS, and the official 
+announcement was made on January 24 2017. Note that BAS is also a distribution institution.
+
 ## Distribution Institution(s)
 
-The distribution institution is the University of Saskatchewan. See Appendix C for a description
-of the current system in place at the University of Saskatchewan, and a brief description of the 
-previous system.
+The distribution institutions are currently USASK and BAS. See Appendix C for a description
+of the current system in place at USASK, and a brief description of the 
+previous system. See Appendix D for a description of the current system in place at BAS.
 
 ## Membership
 
-The membership of the working group should consist of several types of members:
+The membership of the working group consists of several types of members:
 
-1.  Those responsible for providing data to the holding institution.
-2.  Those responsible for accessing data from data mirrors to download to their PI institution.
-3.  Those responsible for receiving hard drives from the four month distribution.
-4.  Others who wish to observe.
+1. The chair or co-chairs
```

Comment:
Where do people like Marina, Paul fit in here? They help maintain the system, so that should be indicated somehow in the responsibilities list

Reply:
Maybe remove these 4 points completely, and reword the responsibilities list.